python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -623,89 +668,176 @@ def load_assets_from_dbt_manifest(
     node_info_to_definition_metadata_fn: Callable[
         [Mapping[str, Any]], Mapping[str, MetadataUserInput]
     ] = default_metadata_fn,
-    display_raw_sql: Optional[bool] = None,
-    dbt_resource_key: str = "dbt",
 ) -> Sequence[AssetsDefinition]:
     """Loads a set of dbt models, described in a manifest.json, into Dagster assets.
 
     Creates one Dagster asset for each dbt model. All assets will be re-materialized using a single
     `dbt run` command.
 
     Args:
-        manifest_json (Optional[Mapping[str, Any]]): The contents of a DBT manifest.json, which contains
+        manifest (Optional[Mapping[str, Any]]): The contents of a DBT manifest.json, which contains
```

Comment:
We should support `os.PathLike` here as well, for parity with `@dbt_assets`. This can happen in a separate PR though.

Reply:
👍